great-idea/css/index.css

```diff
@@ -62,3 +62,141 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+
+/* ~~~~~~~~~~~~~~~ My Code ~~~~~~~~~~~~~~~ */
+
+.container {
+	display: flex;
+	
+	margin-top: 20px;
```

Comment:
Personally, I would have made these margins a little bigger. My personal rule of thumb for pages like this is that the biggest whitespace should be the margin around the edge of the page. If you have larger gaps than that between pieces of content, it can feel a little disjointed.